_api-reference/cat/cat-allocation.md

```diff
@@ -27,6 +27,40 @@ GET /_cat/allocation/{node_id}
 ```
 <!-- spec_insert_end -->
 
+<!-- spec_insert_start
+api: snapshot.restore
+component: example_code
+rest: GET _nodes/stats/indices
```

Comment:
Let's keep this cat/allocation example in this PR. I think `rest:' param value is incorrect though